docs/application/tizen-studio/setup/install-sdk.md

```diff
@@ -49,21 +49,23 @@ To install the Tizen Studio using the CLI installer:
 
 2. Run the CLI installer:
 
-   - On Microsoft Windows&reg; the command prompt opens and the installer is automatically executed.
+   - On Microsoft Windows&reg;, the command prompt opens and the installer is automatically executed.
    - On Ubuntu and macOS, open the terminal, go to the directory where the installer is downloaded, and enter the `chmod +x` command to apply the execute permission to the installer file. Then, execute the installer by entering the command with the following syntax:
 
    ```
    web-cli_Tizen_Studio_<version> [options] [<directory path>]
    ```
 
    **Table: Install options**
+
    | Option             | Description                              |
    |--------------------|------------------------------------------|
    | `--show-license`   | Displays the Tizen Studio software license agreement.<br/> You must use this option alone. Do not use with other options. |
    | `--accept-license` | Accepts the license terms.               |
    | `--no-java-check`  | Skips the Java version check.            |
 
    **Table: Install command parameters**
```

Comment:
line67: Is this table okay with the different browser such as firefox?

Reply:
No, it should show same layout on other browsers, too.
I fixed the tag. Would you please check the updated?